chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -17,42 +17,57 @@
 # limitations under the License.
 #
 
+ceph_conf = node[:glance][:rbd][:store_ceph_conf]
+admin_keyring = node[:glance][:rbd][:store_admin_keyring]
+
 ceph_env_filter = " AND ceph_config_environment:ceph-config-default"
 ceph_servers = search(:node, "roles:ceph-osd#{ceph_env_filter}") || []
 if ceph_servers.length > 0
   include_recipe "ceph::keyring"
 else
-  # If external Ceph cluster will be used,
-  # we need install ceph client packages
-  if node[:platform] == "suse" && File.exists?("/etc/ceph/ceph.client.admin.keyring")
+  # If Ceph confifuration file is present,
+  # external Ceph cluster will be used,
+  # we have to install ceph client packages
+  return unless File.exists?(ceph_conf)
```

Comment:
I guess we don't have the same readability guidelines ;-) I'd put an empty line after this new one (because of the `return`).